Client/Nimbus/OnboardingFeatureLayer/NimbusOnboardingFeatureLayer.swift

```diff
@@ -180,4 +185,16 @@ class NimbusOnboardingFeatureLayer: NimbusOnboardingFeatureLayerProtocol {
         case .syncDevicesCtd: return ImageIdentifiers.onboardingSyncCTD
         }
     }
+
+    private func getPopupInfoModel(
+        from data: NimbusInstructionPopup,
+        withA11yID a11yID: String
+    ) -> OnboardingDefaultBrowserInfoModel {
+        return OnboardingDefaultBrowserInfoModel(
+            title: data.title,
+            instructionSteps: data.instructions
+                .map { String(format: $0, AppName.shortName.rawValue)},
```

Comment:
Is there only one `%@` for this string in all locale? 😂

Reply:
😭